ioctl/cmd/ws/wsproject.go

```diff
@@ -178,3 +197,49 @@ func getEventInputsByName(logs []*iotextypes.Log, eventName string) (map[string]
 
 	return inputs, nil
 }
+
+// upload content to endpoint, returns fetch url, content hash and error
+func upload(endpoint string, filename, hashstr string) (url string, hash256b *hash.Hash256, err error) {
+	// read file content
+	var content []byte
+	content, err = os.ReadFile(filename)
+	if err != nil {
+		err = errors.Wrap(err, errProjectConfigReadFailed.Error())
+		return
+	}
+
+	// calculate and validate hash
+	hash256b = new(hash.Hash256)
+	*hash256b = hash.Hash256b(content)
```

Comment:
suggest to use unnamed return `(string, *hash.Hash256, error)`
so can avoid the awkward programming here